tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -412,7 +413,15 @@ public function adminCreatesUserUsingTheProvisioningApi(?string $user):void {
 	 * @throws Exception
 	 */
 	public function userHasBeenCreatedOnDatabaseBackend(?string $user):void {
-		$this->adminCreatesUserUsingTheProvisioningApi($user);
+		$this->createUser(
+			$user,
+			null,
+			null,
+			null,
+			true,
+			'api'
+		);
```

Comment:
This function returns som response why not check the status code as well :smiley: ?

Reply:
this was deleted after rebase due to changes from this pr https://github.com/owncloud/ocis/pull/7019